plugins/importexport/crossref/templates/index.tpl

```diff
@@ -0,0 +1,113 @@
+	{assign var="allowExport" value=false}
+{else}
+	{assign var="allowExport" value=true}
+{/if}
+
+<script type="text/javascript">
+	// Attach the JS file tab handler.
+	$(function() {ldelim}
+		$('#importExportTabs').pkpHandler('$.pkp.controllers.TabHandler');
+		//$('#importExportTabs').tabs('option', 'cache', true);
+	{rdelim});
+</script>
+<div id="importExportTabs">
+	<ul>
+		<li><a href="#settings-tab">{translate key="plugins.importexport.common.settings"}</a></li>
+		{if $allowExport}
+			{if $exportArticles}
+				<li><a href="#exportSubmissions-tab">{translate key="plugins.importexport.common.export.articles"}</a></li>
+			{/if}
+			{if $exportIssues}
+				<li><a href="#exportIssues-tab">{translate key="plugins.importexport.common.export.issues"}</a></li>
+			{/if}
+		{/if}
+	</ul>
+	<div id="settings-tab">
+		{if !$allowExport}
+			<div class="pkp_notification" id="crossrefConfigurationErrors">
+				{foreach from=$configurationErrors item=configurationError}
+					{if $configurationError == $smarty.const.DOI_EXPORT_CONFIG_ERROR_DOIPREFIX}
+						{include file="controllers/notification/inPlaceNotificationContent.tpl" notificationId=crossrefConfigurationErrors notificationStyleClass=notifyWarning notificationTitle="plugins.importexport.common.missingRequirements"|translate notificationContents="plugins.importexport.common.error.DOIsNotAvailable"|translate}
+					{elseif $configurationError == $smarty.const.DOI_EXPORT_CONFIG_ERROR_SETTINGS}
+						{include file="controllers/notification/inPlaceNotificationContent.tpl" notificationId=crossrefConfigurationErrors notificationStyleClass=notifyWarning notificationTitle="plugins.importexport.common.missingRequirements"|translate notificationContents="plugins.importexport.common.error.pluginNotConfigured"|translate}
+					{/if}
+				{/foreach}
+				{if !$currentContext->getSetting('publisherInstitution')|escape}
+					{include file="controllers/notification/inPlaceNotificationContent.tpl" notificationId=crossrefConfigurationErrors notificationStyleClass=notifyWarning notificationTitle="plugins.importexport.common.missingRequirements"|translate notificationContents="plugins.importexport.crossref.error.publisherNotConfigured"|translate}
+				{/if}
+				{if !$exportArticles && !$exportIssues}
+					{include file="controllers/notification/inPlaceNotificationContent.tpl" notificationId=crossrefConfigurationErrors notificationStyleClass=notifyWarning notificationTitle="plugins.importexport.common.missingRequirements"|translate notificationContents="plugins.importexport.crossref.error.noDOIContentObjects"|translate}
+				{/if}
+			</div>
+		{/if}
+
+		{url|assign:crossrefSettingsGridUrl router=$smarty.const.ROUTE_COMPONENT component="grid.settings.plugins.settingsPluginGridHandler" op="manage" plugin="CrossRefExportPlugin" category="importexport" escape=false}
+		{load_url_in_div id="crossrefSettingsGridContainer" url=$crossrefSettingsGridUrl}
+	</div>
+
+	{if $allowExport}
```

Comment:
Reduce indentation by one tab (since this is getting pretty deeply nested):

    {if $allowExport && exportArticles}